package.json

```diff
@@ -36,11 +36,12 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@asyncapi/generator": "^1.9.3",
+    "@asyncapi/glee": "^0.10.23",
```

Comment:
You don't need Glee here. Actually, you're installing Glee on Glee 😅 so not sure how this would behave hehe!

Reply:
ohkk that happened bymistakely ig when I was trying to run examples in the start😅, but no worries I removed that